docs/Advanced-Topics.rst

```diff
@@ -16,7 +16,9 @@ Categorical Feature Support
 ---------------------------
 
 -  LightGBM offers good accuracy with integer-encoded categorical features. LightGBM applies
-   `Fisher (1958) <https://www.jstor.org/stable/2281952>`_  to find the optimal split over categories,
+   `Fisher (1958) <http://www.csiss.org/SPACE/workshops/2004/SAC/files/fisher.pdf>`_
+   to find the optimal split over categories as
+   `described here <http://lightgbm.readthedocs.io/en/latest/Features.html#optimal-split-for-categorical-features>`_,
```

Comment:
Please use relative links where possible. This is needed because users may browse docs at both GitHub and RTD.

`./Features.html#optimal-split-for-categorical-features`